src/register-new-subscriber.js

```diff
@@ -65,11 +90,10 @@ exports.registerNewSubscriber = async (event) => {
               }`,
           QueueUrl: entitlementQueueUrl,
         };
-
         await sqs.sendMessage(SQSParams).promise();
       }
-
-      return lambdaResponse(200, 'Success! Registration completed. You have purchased an enterprise product that requires some additional setup. A representative from our team will be contacting you within two business days with your account credentials. Please contact Support through our website if you have any questions.');
+       await setBuyerNotificationHandler(contactEmail);
```

Comment:
This step should be performed only if MarketplaceSellerEmail is provided. MarketplaceSellerEmail should be optional.